be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -232,6 +244,9 @@ class FileColumnIterator : public ColumnIterator {
 
     // current rowid
     rowid_t _current_rowid = 0;
+
+    // page indexes those are DEL_PARTIAL_SATISFIED
+    std::unordered_set<uint32_t> _delete_partial_statisfied_pages;
```

Comment:
`unordered_set` here seems to be overkill, isn't `std::vector<uint8_t> _page_delete_status` simpler to deal with?

Reply:
ok